ansible/polipo/tasks/polipo.yml

```diff
@@ -0,0 +1,28 @@
+---
+- name: create polipo dir
+  file:
+    path: /home/vagrant/polipo
+    state: directory
+    owner: vagrant
+    group: vagrant
+
+- name: copy template files
+  copy:
+    src: "{{ item }}"
+    dest: /home/vagrant/polipo
+    owner: vagrant
+    group: vagrant
+  with_items:
+    - Dockerfile
+    - kitchen-template.yml
+    - kitchen.local.yml
+
+- name:
+  command: docker build -t="docker-polipo" .
```

Comment:
Plase, us [Docker](http://docs.ansible.com/docker_image_module.html) module for Ansible